### DISCUSSION TITLE
Don't have a University email for requesting a student discount

### DISCUSSION THREAD

**Discussion by jgonzalezalbisu** [community/community#53695] [GitHub Education]
Hi,

I don't have a university email for requesting a student discount where I live. I'm an alumni and current supporter of some topics given at the institute. The university is UDE (Universidad De la Empresa), located in 21 de Setiembre 2741, 11300 Montevideo, Uruguay.

The past year when I requested the student discount in https://education.github.com/discount_requests/application, after I selected my university and my personal email I was not requested to have a @[ude.edu.uy](http://ude.edu.uy/) domain email, as students and alumni don't have one, so I could request the discount without any problem.

Last week when I was requested to confirm again my information I realized than when selecting my personal email I was not able to continue after selecting the UDE university as I was requested to have an email with the corresponding domain, which was not happening until last year. 

This ticket is for letting you know that the students/teachers/almuni of the university don't have a @[edu.ude.uy](http://edu.ude.uy/) email because of the university policy.

Can this be resolved?

Thanks in advance

**Comment by Irian** [comment]
I'm facing the same issue, which is strange as the documentation says that a prerequisite is to 'Have a verifiable school-issued email address **or upload documents that prove your current student status**'. I cannot reach the step to upload the documents as the system requires a school-issued email address, but I don't get one from my university.

**Reply by jgonzalezalbisu** [reply]
@Irian actually you can continue to the next screen where you're able to upload documents if you enter the name of your university but do not select any of the ones from the drop down.

**Reply by Irian** [reply]
Sadly I can't. The warning that is shown is:
```
We require applicants of Open University of the Netherlands to use one of these school-issued email addresses to apply:
- ou.nl
- student.ou.nl
- studie.openuniversiteit.be
- studie.ou.nl
-

Please select a different email address.
```

Even when I cheat the form by enabling the continue button using the code editor, I receive a warning saying:

```
Please select the email address you use for school -- ideally one issued by your school.
```

**Comment by ZamanMehdi** [comment]
I am facing the same problem, my university has not provided me with the university domain email and I am not able to submit the request to access the student developer pack.

**Comment by Lzzzs** [comment]
me too

**Comment by sharadcodes** [comment]
I am facing the same problem, my university has not provided me with the university domain email.

**Comment by Zweistein2** [comment]
same here. No email provided by my university and my uploaded documents to proof my enrollment get rejected instantly.
